lib/features/sessions/cubit/bookmark_session_cubit.dart

```diff
@@ -34,6 +35,20 @@ class BookmarkSessionCubit extends Cubit<BookmarkSessionState> {
         bookmarkStatus: bookmarkStatus == BookmarkStatus.bookmarked,
       );
 
+      if (bookmarkStatus == BookmarkStatus.bookmarked) {
+        final session = await _dBRepository.getSession(sessionId);
+        final endTime = session?.endDateTime;
+        final notifcationTime = endTime?.subtract(const Duration(minutes: 5));
+
+        await NotificationService().createScheduledNotification(
```

Comment:
Inject the notification service via the controller. Steps:

- On `lib/features/sessions/cubit/bookmark_session_cubit.dart:20`, add `late NotificationService _notificationService;`
- Add it as a required parameter on the constructor on `lib/features/sessions/cubit/bookmark_session_cubit.dart:11-14`
- Assign it in a similar fashion to `_apiRepository` & `_dBRepository`
- Then ensure to load it in properly in `lib/bootstrap.dart:116`